Ladybird/Qt/BrowserWindow.cpp

```diff
@@ -152,6 +152,15 @@ BrowserWindow::BrowserWindow(Vector<URL::URL> const& initial_urls, WebView::Cook
 
     edit_menu->addSeparator();
 
+    m_find_in_page_action = new QAction("&Find in Page", this);
```

Comment:
This action text this should have an ellipsis ("...") to indicate that it requires additional input before any action is taken.